polynote-kernel/src/main/scala/polynote/kernel/util/RuntimeSymbolTable.scala

```diff
@@ -14,7 +14,7 @@ import scala.collection.mutable
 import scala.collection.JavaConverters._
 import scala.tools.nsc.interactive.Global
 
-final class RuntimeSymbolTable(
+/*final class RuntimeSymbolTable(
```

Comment:
Oops, I forgot to actually delete this file! It's just commented out. Will delete.